virttest/libvirt_vm.py

```diff
@@ -1071,6 +1071,26 @@ def set_kernel_console(self, device, speed=None, remove=False):
         logging.debug("Set kernel params for %s failed.", device)
         return False
 
+    def get_grub_file(self, session):
+        """
+        Find grub file of vm.
```

Comment:
For docstring, first line is the description, then you need one empty line following, then details on param and return.